packages/asc-ui/src/components/ShowMoreShowLess/ShowMoreShowLess.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import ShowMoreShowLess from './ShowMoreShowLess'
+
+describe('ShowMoreShowLess', () => {
```

Comment:
Should we not be testing more functionality?

Reply:
The only logic I want to test is this:
```
useEffect(() => {
    const el = ref.current
    if (el) {
      setShowMoreShowLess(el.clientHeight < el.scrollHeight)
    }
  }, [])
```
The problem is that I can't figure out how to get the `el.clientHeight` and `el.scrollHeight`, as they are alway 0. Also tried to mock useRef, but that doesn't work either...